modules/test-structure/save_test_data.go

```diff
@@ -206,10 +205,16 @@ func LoadTestData(t *testing.T, path string, value interface{}) {
 
 // IsTestDataPresent returns true if a file exists at $path and the test data there is non-empty.
 func IsTestDataPresent(t *testing.T, path string) bool {
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			return false
```

Comment:
Good catch! Should we use `files.FileExists` instead (https://github.com/gruntwork-io/terratest/blob/master/modules/files/files.go#L11) here? The only difference is that it is not explicitly checking for `os.IsNotExist`, so it can also return `false` for other kinds of errors like permission failures. In most cases that error will surface at a later point in the test, but we should probably add:
```
func FileExistsE(path string) (bool, error) {
	_, err := os.Stat(path)
	if err != nil && !os.IsNotExist(err) {
		return false, err
	}
	return err == nil, nil
}
```

Can you update this PR to include that function and use that here?